openapi/src/parts/controlled_aws_storage_folder.yaml

```diff
@@ -0,0 +1,130 @@
+paths:
+  /api/workspaces/v1/{workspaceId}/resources/controlled/aws/storageFolder:
```

Comment:
Re: https://github.com/DataBiosphere/terra-workspace-manager/pull/1198#issuecomment-1511871242

I would be open to renaming this. Considerations -
1) Recommendation from Dan in a previous PR (+ potential other considerations) to avoid using copyrighted or marketing names like 'S3' or 'sagemaker'
2) kept in line with Azure Storage container (more recent than GCP)